core/src/autogluon/core/_setup_utils.py

```diff
@@ -10,7 +10,7 @@
     os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '..', '..')
 )
 
-PYTHON_REQUIRES = '>=3.6, <3.8'
+PYTHON_REQUIRES = '>=3.6, <=3.8'
```

Comment:
I don't think we support 3.8. Is it necessary for this contribution? This kind of change should really be left to a separate PR dedicated to enabling 3.8 support.

Reply:
Not necessary, I enabled it for the purpose of working around the installation on Conda, which will install python3.8 by default: https://docs.conda.io/en/latest/miniconda.html